sites/main-site/src/content/events/2025/hackathon-march-2025/university-of-miami.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Hackathon - March 2025 (University of Miami)'
+subtitle: 'Local node of the nf-core hackathon at the University of Miami, Florida, USA'
+shortTitle: '🇺🇸 University of Miami'
+type: 'hackathon'
+startDate: '2025-03-24'
+startTime: '09:00+00:00'
```

Comment:
This is UTC + offset and therefore currently is set to British time. Please add the correct offset to the start and the end time.

Reply:
Fixed locally. Couldn't push to this PR's branch but pushed to the main repo to merge.